server/routes/profile.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const multer = require("multer");
+const router = express.Router();
+const protect = require("../middleware/auth");
+const { savePhoto } = require("../controllers/profile");
+
+const upload = multer({dest: './uploads'});
+
+// router.route("/photo").post(protect, savePhoto);
+router.route("/savePhoto")
```

Comment:
Please use hyphenated case for route names with multiple words :)

Eg: `edit-profile`